### DISCUSSION TITLE
Is it possible  to also download the text chat in voice channel?

### DISCUSSION THREAD

**Discussion by cloudforst** [Tyrrrz/DiscordChatExporter#873] [Q&A]
Since the discord has updated and included new function.
if it possible to download the text chat in voice channel as well?
Thanks!

**Comment by Tyrrrz** [comment]
I'm not familiar with how it's implemented in Discord yet, but I'm willing to assume it's just a regular channel under the wraps. If you can get its ID somehow, you will be able to use the CLI version to download it.

**Reply by 96-LB** [reply]
Yup, this is correct. Tested it out with CLI a few weeks ago and it exported perfectly. We might want to make voice channels visible in the GUI now since the ID is shared between the voice and text channel.

**Reply by Tyrrrz** [reply]
Are the voice-text channels permanent or are they similar to threads where they archive and new ones are spun up?

**Reply by 96-LB** [reply]
Permanent, as far as I can tell.

**Reply by cloudforst** [reply]
Thanks for all the reply, it's really helpful.
it seems that if I can get "the voice channel ID" then I could successfully export the chat log. 
(I used to find my discord "token" at the web and apply to the exporter)
But, where can I find the ID in my voice channel?
(Maybe the token of the voice channel? I am not sure about how it works) 
(Sorry that I have little knowing about coding or web constructure)
Can someone teach me how it works? Thank you

**Reply by 96-LB** [reply]
You can follow the instructions [here](https://github.com/Tyrrrz/DiscordChatExporter/wiki/Obtaining-Token-and-Channel-IDs#how-to-get-a-server-id-or-a-server-channel-id) to copy a channel ID. It works the same whether you use a text or a voice channel.

**Reply by Tyrrrz** [reply]
Related: https://github.com/Tyrrrz/DiscordChatExporter/issues/874
